src/Modules/Admin/Booking/BookingList.tsx

```diff
@@ -1,7 +1,189 @@
-import styles from './BookingList.module.css';
+import {
+  Box,
+  Stack,
+  Typography,
+  Table,
+  TableBody,
+  TableCell,
+  tableCellClasses,
+  TableContainer,
+  TableHead,
+  TableRow,
+  Paper,
+  Skeleton,
+  Alert,
+  Pagination,
+} from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { Visibility } from '@mui/icons-material';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { axiosInstanceAdmin, BOOKING_URLS } from '../../../Services/END_POINTS/ADMIN/URLS';
+const StyledTableCell = styled(TableCell)(() => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: 'rgba(226, 229, 235, 1)',
+    color: 'rgba(30, 38, 62, 1)',
+  },
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+    color: '#333',
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(() => ({
+  '&:nth-of-type(odd)': {
+    backgroundColor: 'rgba(255, 255, 255, 1)',
+  },
+  '&:nth-of-type(even)': {
+    backgroundColor: 'rgba(248, 249, 251, 1)',
+  },
+  '&:last-child td, &:last-child th': {
+    border: 0,
+  },
+}));
+
+interface Room {
+  roomNumber: string;
+}
+
+interface User {
+  userName: string;
+}
+
+interface Booking {
+  _id: string;
+  room: Room;
+  user: User;
+  startDate: number;
+  endDate: number;
+  totalPrice: number;
+}
+ const renderSkeletonRows = () =>
```

Comment:
We can capitalize the first letter to render it as a normal JSX component instead of needing to call it as a function. I suggest renaming it to `SkeltonRows`